dataScrape/src/pokemonTableUpdate.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+class pokemonTableUpdate
+{
+    private $db;
+    private $pokemonDataList;
+
+    /**
+     * pokemonTableUpdate constructor.  Opens pokedex db connection
+     *
+     * @param array $pokemonDataList contains an array for each pokemon
+     * @param string $dbServerIP is the IP address of the database server
+     */
+    public function __construct(array $pokemonDataList, string $dbServerIP)
+    {
+        $this->db = new PDO('mysql:dbname=pokedex_hb;host=' . $dbServerIP, 'root');
+        $this->db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+        $this->pokemonDataList = $pokemonDataList;
+    }
+
+    /**
+     * Adds values from each array to pokemon table in db
+     */
+    public function dbUpdate()
+    {
+        foreach ($this->pokemonDataList as $pokemonData) {
+            $id = $pokemonData['id'];
```

Comment:
why assign the array lkeys to variables? just use the array in the execute?

Reply:
changed in b1574c5